src/app/pages/forms-and-validation/in-a-list/in-a-list.component.scss

```diff
@@ -0,0 +1,38 @@
+@import '~@pxblue/colors/palette.scss';
+
+::ng-deep .pxb-drawer-layout-nav-content {
+    background-color: map-get($pxb-white, 50);
+}
+section.mat-typography {
+    padding-top: 24px;
+    max-width: 686px;
```

Comment:
I've been seeing a lot of these max-width settings to arbitrary numbers. I know the design shows 686px, but it would be nice to at least have this be a multiple of 8px or possibly come up with a standard for how wide we want the content to be so it's not a different value for each example. 

@huayunh is there a reasonable max-width standard we can have for these examples on desktop?

If Huayun is cool with this feel free to ignore this comment.